meetings/__init__.py

```diff
@@ -213,6 +220,16 @@ async def log_message(self, evt: MessageEvent, match: Tuple[str]) -> None:
         await self.log_tag("topic", evt)
         #await evt.react("✏️️")
 
+      # Change the meetingname
+      if re.search("\^meetingname", evt.content.body):
+        meetingname = evt.content.body.removeprefix("^meetingname").strip()
+        if meetingname:
+          await self.change_meetingname(meetingname, evt)
```

Comment:
I feel this should be a !command and have a powerlevel check, not everyone should be able to set a meetingname. Thoughts?

Reply:
I'd also consider `topic` to potentially be a protected command too...

Reply:
ok -- will add a issue for this one -- it is also kind of tied into the idea of chairs as well, which also needs to be implemented.